model/project_ref.go

```diff
@@ -1489,3 +1440,92 @@ func GetUpstreamProjectName(triggerID, triggerType string) (string, error) {
 	}
 	return upstreamProject.DisplayName, nil
 }
+
+func pipelineForValueIsBool(projectKey, repoKey string, val bool) []bson.M {
```

Comment:
Could you rename these functions so that it's clear they're for project refs? Since there are a bunch of different models in this package it'll be nice to be able to immediately see what these are by name